cmd/download.go

```diff
@@ -59,21 +59,21 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	if err != nil {
 		return err
 	}
-	exercise, err := flags.GetString("exercise")
+	slug, err := flags.GetString("exercise")
 	if err != nil {
 		return err
 	}
-	if uuid == "" && exercise == "" {
+	if uuid == "" && slug == "" {
 		return errors.New("need an --exercise name or a solution --uuid")
 	}
 
-	var slug string
+	var param string
```

Comment:
What is param supposed to represent an exercise `iteration`?

Reply:
param is the param to the URL that we're calling on the API. It's either the string `latest`, or an actual UUID.